lib/training/wiki_training_loader.rb

```diff
@@ -35,10 +35,9 @@ def load_from_wiki
                           level: 'info', extra: { wiki_pages: source_pages }
 
     thread_count = [CONCURRENCY, source_pages.count].min
-    threads = source_pages.in_groups(thread_count, false).map.with_index do |wiki_page_group, i|
-      Thread.new(i) { add_trainings_to_collection(wiki_page_group) }
+    source_pages.in_groups(thread_count, false).each do |wiki_page_group|
```

Comment:
Since this gets reid of the conccurency altogether, there's some additional cleanup above here that needs to be done. The constant and thread_count is now useless.